apps/ts.c

```diff
@@ -888,7 +889,8 @@ static TS_VERIFY_CTX *create_verify_ctx(const char *data, const char *digest,
         f = TS_VFY_VERSION | TS_VFY_SIGNER;
         if (data != NULL) {
```

Comment:
Push the `out` declaration and init right here, inside the `if` block 
`BIO *out = BIO_new_file(data, "rb");`